book-example/src/format/config.md

```diff
@@ -24,66 +20,118 @@ additional-css = ["custom.css"]
 It is important to note that **any** relative path specified in the in the configuration will
 always be taken relative from the root of the book where the configuration file is located.
 
-### General metadata
 
-- **title:** The title of the book
-- **author:** The author of the book
-- **description:** A description for the book, which is added as meta information in the html `<head>` of each page
-
-**book.toml**
-```toml
-title = "Example book"
-author = "John Doe"
-description = "The example book covers examples."
-```
+### General metadata
 
-Some books may have multiple authors, there is an alternative key called `authors` plural that lets you specify an array
-of authors.
+This is general information about your book.
 
-**book.toml**
-```toml
-title = "Example book"
-authors = ["John Doe", "Jane Doe"]
-description = "The example book covers examples."
-```
-
-### Source directory
-By default, the source directory is found in the directory named `src` directly under the root folder. But this is configurable
-with the `source` key in the configuration file.
+- **title:** The title of the book
+- **authors:** The author(s) of the book
+- **description:** A description for the book, which is added as meta
+  information in the html `<head>` of each page
+- **src:** By default, the source directory is found in the directory named
+  `src` directly under the root folder. But this is configurable with the `src`
+  key in the configuration file.
+- **build-dir:** The directory to put the rendered book in. By default this is
+  `book/` in the book's root directory.
 
 **book.toml**
 ```toml
+[book]
 title = "Example book"
 authors = ["John Doe", "Jane Doe"]
 description = "The example book covers examples."
-
-source = "my-src"  # the source files will be found in `root/my-src` instead of `root/src`
+src = "my-src"  # the source files will be found in `root/my-src` instead of `root/src`
+build-dir = "build"
```

Comment:
You moved the `build-dir` into the common configuration. Hmmm. If I have multiple output generators, I cannot put them into different directorys. I think the other way was better. What are the arguments for doing it this way?

Reply:
My thoughts were that (in the long term) you would have a single `build/` directory where *all* renderers would put their rendered output. During the rendering stage a particular renderer is simply told the directory they should put things in, instead of each renderer doing their own thing and deciding it themselves. 

That way `mdbook` would ensure you have a standardized directory layout looking something like this:

- my_book
  - build
    - html
    - pdf
    - some_other_renderer
  - src
  - book.toml

With the special case being that if you're only using one renderer, `mdbook` will put tell a renderer to put its stuff directly in the root (avoiding the redundant extra level of nesting from `build/html/`).

Reply:
ok, that sounds fine :)